nanvm-lib/src/js/number.rs

```diff
@@ -49,7 +49,7 @@ pub mod test {
         assert!(is_valid(INFINITY));
         assert!(is_valid(NAN));
         assert!(is_valid(NEG_INFINITY));
-        assert_eq!((0.0f64 / 0.0).to_bits(), NAN);
```

Comment:
I would keep both, (0.0f64 / 0.0) and f64::NAN. The test checks that there is only one NaN value.